sdk/storage/azure-storage-blobs/src/blob_responses.cpp

```diff
@@ -39,12 +38,13 @@ namespace Azure { namespace Storage { namespace Blobs {
   {
     while (true)
     {
-      auto rawResponse = PollInternal(context);
+      // Poll will update the raw response.
+      Poll(context);
 
       if (m_status == Azure::Core::OperationStatus::Succeeded)
       {
         return Azure::Response<Models::GetBlobPropertiesResult>(
-            m_pollResult, std::move(rawResponse));
+            m_pollResult, std::move(m_rawResponse));
```

Comment:
Ah, I get it. We have to make a copy here.

Reply:
Cannot use move, or the raw response in `Operation<T>` will become empty

Reply:
hahahaha,,,   I just removed the copy cttor and code for that hahahahha

I think we can add it later, as it is implementation detail and won't impact/break customers